composer.json

```
@@ -23,7 +23,7 @@
   },
   "require": {
     "php": ">=5.4",
-    "shopgate/cart-integration-sdk" : "~2.9.70"
+    "shopgate/cart-integration-sdk" : "~2.9.91"
```

For PHP 8 compatibility the SDK should also be upgraded to the latest version 2.10.3. This should be a seamless upgrade for the Gambio plugin.

```suggestion
    "shopgate/cart-integration-sdk" : "~2.10.3"
```

---

I had to update this when fixing CI, so this is resolved now.